eng/pipelines/extra-platforms/runtime-extra-platforms-wasm.yml

```diff
@@ -105,16 +105,18 @@ jobs:
       scenarios:
         - WasmTestOnNodeJS
 
-  # Library tests - Windows
+  # Library tests - Windows - this only runs on the extra pipeline
   - template: /eng/pipelines/common/templates/wasm-library-tests.yml
     parameters:
       platforms:
         - browser_wasm_win
+      nameSuffix: _NodeJs
+      extraBuildArgs: /p:AotHostArchitecture=x64 /p:AotHostOS=$(_hostedOS)
       # Don't run for rolling builds, as this is covered
       isExtraPlatformsBuild: ${{ parameters.isExtraPlatformsBuild }}
       isWasmOnlyBuild: ${{ parameters.isWasmOnlyBuild }}
+      alwaysRun: ${{ parameters.isWasmOnlyBuild }}
       scenarios:
-        - WasmTestOnChrome
```

Comment:
https://github.com/dotnet/runtime/blob/53be9e5a36fa008406eed8720fc72b8e02f21256/eng/pipelines/runtime.yml#L827

Reply:
And this one was not running at all anyway, I suspect it's because we had name duplicate, `runtime` lane was running `runtime (Build browser-wasm windows Release LibraryTests)` and then triggering extra platforms did not add another run of the same name that would have 2 hosts, it was just ignored.